topics/admin/tutorials/interactive-tools/slides.html

```diff
@@ -67,13 +98,30 @@
   - Pros: Needs no special credentials (can be shared)
   - Cons: Requires *wildcard* DNS entry and *wildcard* SSL certificate (not possible at many sites)
 
+???
+
+ - Diverse infrastructures behind Galaxy Interactive Environments and Galaxy Interactive Tools induces various constraints and benefits.
+ - Particularly in building the path between the browser and the container.
+
 ---
 # Anatomy of a running Interactive Tool
 
 ![Galaxy Interactive Tools Proxy Diagram](../../images/interactive-tools/gxit-proxy-diagram.png "Galaxy Interactive Tools Proxy Diagram")
 
 ???
-The source for this figure can be found at: https://docs.google.com/presentation/d/1_4PtfM6A4mOxOlgGh6OGWvzFcxD1bdw4CydEWtm5n8k/
+
+ - This slide illustrates the steps allowing a client to interact with a Galaxy Interactive Tool.
+ - We consider a Galaxy server running behind a reverse proxy (NGINX). 
+ - The client only ever speaks to NGINX on the Galaxy server, running on the standard HTTPS 443 port.
+ - Based on the elements provided by the URL, NGINX redirects the requests to Galaxy over a unix domain socket as usual.
+ - On the other hand, HTTP requests targetting interactive tools are redirected by NGINX to a proxy, called GIE Proxy, running on port 8000.
+ - At this point, remember, when an docker container starts, it will be assigned a host (server or node) and a random port on that host (in our example, port 32768). 
+ - This information is stored by Galaxy in a sqlite database.
+ - So the GIE proxy checks in this sqlite database to know which node and port the IT container is to be found.
+ - The GIE proxy forwards the HTTP request to Docker on that node and port.
+ - Docker, in turn, forwards it to the application (e.g. Jupyter) on the container’s port, defined in the wrapper.
+
+ - The source for this figure can be found at: https://docs.google.com/presentation/d/1_4PtfM6A4mOxOlgGh6OGWvzFcxD1bdw4CydEWtm5n8k/ .
```

Comment:
Could you move this into the slide itself? I don't want polly to try and speak the URL.

Reply:
Done. Tried something with `.reduce70[.footnote[`

Reply:
That's perfect.